akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -373,15 +373,19 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
   }
 
   def uninitialized: Actor.Receive = ({
-    case InitJoin                          ⇒ sender() ! InitJoinNack(selfAddress)
+    case InitJoin ⇒
+      logInfo("Received InitJoin message from [{}] to [{}]", sender(), selfAddress)
```

Comment:
`logInfo` already includes a prefix containing the address of this node, so the `selfAddress` param is not needed